documentation/src/docs/deployment/kubernetes-install.md

```diff
@@ -31,22 +31,28 @@ You may obtain a copy of the License at
 
 ## Requirements
 
-- Kubernetes 1.8+
-- An existing Apache Zookeeper 3.5 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
-- [Pravega Operator](https://github.com/pravega/pravega-operator/edit/master/README.md) manages Pravega clusters deployed to Kubernetes and automates tasks related to operating a Pravega cluster.
-
+- Kubernetes 1.15+
+- Helm 3.2.1+
+- An existing Apache Zookeeper 3.6.1 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
+- An existing Apache Bookkeeper 4.9.2 cluster. This can be easily deployed using our [Bookkeeper Operator](https://github.com/pravega/bookkeeper-operator).
```

Comment:
Bookkeeper version should be upgraded as well, right? We are now using version 4.11. @SrishT what is the minimum version supported in the Bookkeeper Operator?

Reply:
https://github.com/pravega/pravega-operator/blob/master/README.md, here I see 4.9.2 as minimum so I added it. Let me see what version Srishti recommends

Reply:
Ok, would it be more accurate to use `4.9.2 (or higher)`?

Reply:
Sure, I can make that change